esmvalcore/cmor/_fixes/cesm/cesm2.py

```diff
@@ -68,7 +68,7 @@ def _fix_time(self, cube):
         """
         # Only modify time points if data contains a time dimension, is monthly
         # data, and does not describe point measurements.
-        if 'time' not in self.vardef.dimensions:
+        if not self.vardef.has_dimension_that_startswith('time'):
```

Comment:
Maybe it would be better to look up the standard name of the dimension, e.g: `time1` has standard name `time`: https://github.com/ESMValGroup/ESMValCore/blob/576a5bb3a57678ec8c9a5fcf89fc62d5d1ef9c4b/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_coordinate.json#L2213-L2214

Reply:
Thanks @bouweandela, that's a really good suggestion and makes much more sense! I completely missed that there is a `VariableInfo.coordinates` attribute that already stores this information about the `standard_name`. I changed it in https://github.com/ESMValGroup/ESMValCore/pull/1971/commits/3f9300de61834e1ab372eb806b0db1b40157fe80.